### PR TITLE
Fix active link on help

### PR DESCRIPTION
**Pull request by anithafuturefinance**

### CODE REVIEW

**Review by gabriel-ff — APPROVED** [review]

File: templates/header.hbs [hunk 1]
```diff
@@ -16,7 +16,7 @@
           data-label-3="Student Money"
           data-target-3="_blank"
         ></div>
-        <a href="{{settings.base_url}}/help/">Help</a>
+        <a href="https://help.futurefinance.com/hc/en-us">Help</a>
```
Comment (gabriel-ff) [inline templates/header.hbs]: should this link have `active` class to make it highlighted?

**Review by gabriel-ff — COMMENTED** [review]
one question

File: templates/header.hbs (hunk 1 above)
Comment (anithafuturefinance) [reply templates/header.hbs]: Thought so, but then wondered if that link should be highlighted if they browse to https://help.futurefinance.com/hc/en-us/categories/360002179260-Guarantors ?

Comment (anithafuturefinance) [reply templates/header.hbs]: I guess I will add class=active

Comment (anithafuturefinance) [reply templates/header.hbs]: Done

**Pull request merged**